Assets/MRTK/Providers/Experimental/WindowsSceneUnderstanding/WindowsSceneUnderstandingObserver.cs

```diff
@@ -632,6 +650,10 @@ private async Task RunObserverAsync(CancellationToken cancellationToken)
                                 sceneObjects.Add(saso.Guid, saso);
                                 SendSceneObjectAdded(saso, saso.Guid);
                             }
+                            else
+                            {
+                                SendSceneObjectUpdated(saso, saso.Guid);
```

Comment:
i believe you will need to update the sceneObjects dictionary

```
sceneObjects[saso.Guid] = saso;
```

Reply:
Totally agreed

Reply:
small nit: i would move that line above the send